client/my-sites/importer/style.scss

```diff
@@ -40,7 +40,7 @@
 	}
 
 	&.medium {
-		background-color: #00ab6c;
+		background-color: #12100e;
```

Comment:
Should this be a variable like the other colors, above?

Reply:
Colours like `--color-wpcom` are WPCOM brand or site colour scheme colours and used widely, but because these others are for third party branding and used only once I'd say they're fine as literal values.